### PR TITLE
Improve/checkout

### PR DESCRIPTION
**Pull request by steveamaza**
Why was this change made?

- Creates a stylized button instead of plain no-style "Pay Now" button
- Improves #4 by optimizing currency conditional
- Bypasses issue where multiple checkouts were being popped up on clicking "Pay Now" button.

### CODE REVIEW

File: modules/gateways/paystack.php [hunk 1]
```diff
@@ -174,34 +176,45 @@ function paystack_link($params)
...
-          callback: function(response){
-            window.location.href = \''.addslashes($callbackUrl).'&trxref=\' + response.trxref;
-          },
-          onClose: function(){
-              paystackIframeOpened = false;
-          }
+            key: \''.addslashes(trim($publicKey)).'\',
+            email: \''.addslashes(trim($email)).'\',
+            phone: \''.addslashes(trim($phone)).'\',
+            amount: '.$amountinkobo.',
```
Comment (ibrahimlawal) [inline modules/gateways/paystack.php]: How come we can remove the `* 100` from this line?

File: modules/gateways/paystack.php [hunk 2]
```diff
@@ -174,34 +176,45 @@ function paystack_link($params)
...
+                $(\'img[alt="Loading"]\').hide();
+                $(\'div.alert.alert-info.text-center\').html(\'Click the button below to retry payment...\');
+                create_button();
             }
        }
+       function create_button(){
+        if(!button_created){
+            button_created = true;
+            $(\'.payment-btn-container2\').append(\'<button type="button"'. 
+                ' onClick="window.location.reload()"'.
```
Comment (ibrahimlawal) [inline modules/gateways/paystack.php]: I see you are reloading the page instead of calling payWithPaystack... What informed this decision?

Comment (steveamaza) [reply modules/gateways/paystack.php]: @ibrahimlawal this is the best way I can think of handling the issue where the checkout is shown twice on top of each other (which renders it unusable)

File: modules/gateways/paystack.php (hunk 1 above)
Comment (steveamaza) [reply modules/gateways/paystack.php]: @ibrahimlawal the multiplication is already done at https://github.com/steveamaza/paystack-whmcs/blob/8a536a3d88a52011d27af256140d04fa6bb0c569/modules/gateways/paystack.php#L114

Comment (ibrahimlawal) [reply modules/gateways/paystack.php]: So, this is a bug-fix then. seen.

**Pull request merged**